android/src/main/java/bbct/android/common/activity/BaseballCardDetails.java

```diff
@@ -222,23 +222,11 @@ public boolean onOptionsItemSelected(MenuItem item) {
             case R.id.save_menu:
                 this.onSave();
                 return true;
-
-            case android.R.id.home:
-                this.onHome();
-                return true;
         }
 
         return false;
     }
 
-    private void onHome() {
-        Fragment list = BaseballCardList.getInstance(null);
-        this.getActivity().getSupportFragmentManager().beginTransaction()
-                .replace(R.id.fragment_holder, list, FragmentTags.CARD_LIST)
-                .addToBackStack(FragmentTags.CARD_LIST)
-                .commit();
-    }
```

Comment:
At one time this was necessary when the app starts with no cards because it jumps directly to `BaseballCardDetails` instead of `BaseballCardList`. This function allowed the app to go "home" to `BaseballCardList` in such a case.

Is this no longer needed with your current implementation of the "up" button?

Reply:
@codeguru42 yup, it won't be needed, as what happens is with or without cards, the BaseballCardList is added first to the Fragment stack, then if the cardslist is empty we go to the BaseballCardDetails.java and add it above the BaseballCardList.java, so when we pop from the stack, we go directly back to the initial Fragment in the Stack.